airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -60,6 +62,13 @@ def _load_body_to_dict(body: str) -> dict:
     return body_dict
 
 
+def should_retry_create_job(exception: BaseException) -> bool:
+    """Check if an Exception indicates a transient error and warrants retrying."""
+    if isinstance(exception, ApiException):
+        return str(exception.status) == "500"
+    return False
+
+
```

Comment:
Could you please move it to utils and import from there?

Reply:
@moiseenkov sure, I have done it